x/oracle/keeper/slash.go

```diff
@@ -55,3 +49,13 @@ func (k Keeper) SlashAndResetMissCounters(ctx sdk.Context) {
 		return false
 	})
 }
+
+func (k Keeper) PossibleWinsPerSlashWindow(ctx sdk.Context) int64 {
```

Comment:
```suggestion
// possibleWinsPerSlashWindow returns xyz
func (k Keeper) possibleWinsPerSlashWindow(ctx sdk.Context) int64 {
```

Reply:
@zarazan I also meant for the keeper function to not be public - I don't think it's necessary

Reply:
I'm using it in the test as well and that is in a separate package `keeper_test`

Reply:
I added a test for this function and updated the SlashWindow comment for clarity